src/Implementation/WarmStorage/WarmStorage/WarmStorageCoordinator.cs

```diff
@@ -56,8 +56,7 @@ private WarmStorageCoordinator(EventProcessorHost host)
             {
                 MaxBatchSize = configuration.MaxBatchSize,
                 PrefetchCount = configuration.PreFetchCount,
-                ReceiveTimeOut = configuration.ReceiveTimeout,
-                InvokeProcessorAfterReceiveTimeout = true
+                ReceiveTimeOut = configuration.ReceiveTimeout
```

Comment:
In previous releases, `ReceiveTimeOut` was ignored unless `InvokeProcessorAfterReceiveTimeout` was set to true. Has this changed in the current release?